cocos2d/core/base-nodes/CCNodeCanvasRenderCmd.js

```diff
@@ -368,24 +371,50 @@ cc.Node.RenderCmd.prototype = {
     },
 
     visitChildren: function(){
```

Comment:
I do not know the relation between this code and batch rendering. Does this solve another issue?